### ISSUE TITLE
FlexFormParsingModifyEventListener fails during EXT:container sorting wizard

### ISSUE THREAD

**Issue by fjacobi** [in2code-de/powermail#1064]
```
Core: Exception handler (WEB): Uncaught TYPO3 Exception: In2code\Powermail\EventListener\FlexFormParsingModifyEventListener::getRequest(): Return value must be of type ?Psr\Http\Message\ServerRequestInterface, none returned | TypeError thrown in file /var/path/domain.de/typo3conf/ext/powermail/Classes/EventListener/FlexFormParsingModifyEventListener.php in line 92. Requested URL: https://www.domain.de/typo3/install.php?install%5Bcontroller%5D=upgrade&install%5Bcontext%5D=backend
```

I guess this needs a CLI check

**Comment by mschwemer** [comment]
Which Powermail and TYPO3 version do you use? 

Is fixed in e69ca4a3a9a2530565e7fa66a04034a449e8da11 for v12

**Comment by mschwemer** [comment]
Checked the dates of commit and issue. Commit was after the issue. So it is fixed ;-)
